app/build.gradle

```diff
@@ -139,7 +139,7 @@ android {
         compose true
     }
     composeOptions {
-        kotlinCompilerExtensionVersion compose_version
+        kotlinCompilerExtensionVersion libs.versions.compose.compiler.get()
```

Comment:
Compose compiler is bound to Kotlin version, not Compose version - see https://developer.android.com/jetpack/androidx/releases/compose-kotlin